tox.ini

```diff
@@ -10,6 +10,8 @@ deps =
 commands =
     coverage run --source={toxinidir}/sphinx_click -m unittest discover -s tests/
     coverage report
+pip_pre =
+    pre: true
```

Comment:
Not sure if this is necessary. If we were testing 3.8, maybe, but these are out now and there's no reason to support anyone using the pre-release version, IMO

Reply:
it is running tests for sphinx 2.0.0b1 before it is released as 2.0.0. I think that is a responsible thing to do, but ultimately, that's your call.

Reply:
Ah. The more you know. Fine by me, so